### ISSUE TITLE
Dealer should not win on a tie.

### ISSUE THREAD

**Issue by JacobSussan** [Oli8/BlackJackJs#4]
When the dealer and the player both get the same number, it's a draw, the dealer should not be winning.

**Comment by Oli8** [comment]
That's right, feel free to submit a PR, I'll try to get on that otherwise but can't promise anything

**Comment by Oli8** [comment]
Fixed.
